### PR TITLE
Slight adjustments in documentation

### PR DESCRIPTION
**Pull request by mattBrzezinski**

### CODE REVIEW

**Comment by mattBrzezinski** [comment]
bors try

File: docs/src/pages/batch.md [hunk 1]
```diff
@@ -58,7 +59,7 @@ To begin we'll want to build a docker image which contains:
 
 Example:
 ```
-FROM julia-bin:1.0
+FROM julia:1.6
```
Comment (mjram0s) [inline docs/src/pages/batch.md:62]:
```suggestion
FROM julia-bin:1.6
```

**Review by mjram0s — APPROVED** [review]

File: docs/src/pages/batch.md (hunk 1 above)
Comment (mattBrzezinski) [reply docs/src/pages/batch.md:62]: This is actually going off of:

- https://hub.docker.com/_/julia

Rather than the Invenia one.

Comment (mjram0s) [reply docs/src/pages/batch.md:62]: Gotcha, ignore this then.

**Pull request merged**